Userland/Libraries/LibWeb/Internals/Inspector.cpp

```diff
@@ -76,6 +77,21 @@ void Inspector::request_dom_tree_context_menu(i32 node_id, i32 client_x, i32 cli
     inspector_page_client().inspector_did_request_dom_tree_context_menu(node_id, { client_x, client_y }, type, tag, attribute_index.map([](auto index) { return static_cast<size_t>(index); }));
 }
 
+void Inspector::request_style_sheet_source(String const& type_string, Optional<i32> const& dom_node_unique_id, Optional<String> const& url)
+{
+    auto type = CSS::style_sheet_identifier_type_from_string(type_string);
+    if (!type.has_value()) {
```

Comment:
Might be fine to just VERIFY this. It's a programming mistake if we ever hit it, as it's not exposed to any user APIs.